indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -41,11 +32,23 @@
     {% endfor %}
 {% endmacro %}
 
-<div class="toolbar">
-    <div class="group right">
-        <a class="i-button js-reset-btn" type="button">
-            {%- trans %}Clear filters{% endtrans -%}
-        </a>
+<div class="action-box">
+    <div class="section">
+        <div class="icon icon-wrench"></div>
+        <div class="toolbar right">
+            <a class="i-button js-reset-btn icon-close warning" type="button">
+                {%- trans %}Clear filters{% endtrans -%}
+            </a>
+        </div>
+        <div class="text">
+            <div class="label">Customize registration list</div>
+            <div>
+                {% set filter_icon = '<i class="icon-filter"></i>' | safe %}
+                {%- trans filter_icon=filter_icon -%}
```

Comment:
`{%- trans -%}` is enough.

You only need to specify variables in the `trans` tag if you do anything fancy with them (access attributes, apply filters, etc)

http://jinja.pocoo.org/docs/dev/templates/#i18n-in-templates

> To translate a template expression — say, using template filters, or by just accessing an attribute of an object — you need to bind the expression to a name for use within the translation block: